layouts/shortcodes/docs/languages/exporters-intro.md

```diff
@@ -1,22 +1,27 @@
-<!-- cSpell:ignore isset cond -->
+{{ $lang := .Get 0 -}}
 
-{{ $lang := .Get 0 -}} {{ $data := index $.Site.Data.instrumentation $lang }}
-{{ $name := cond (isset $.Site.Data.instrumentation $lang) (printf "OpenTelemetry %s" $data.name) "the language specific implementations" -}}
+{{ $data := index $.Site.Data.instrumentation $lang -}}
 
-In order to visualize and analyze your telemetry, you will need to export your
-data to an [OpenTelemetry Collector](/docs/collector/) or a backend such as
+{{ $name := $data.name -}}
+
+To visualize and analyze telemetry, export it to the
+[OpenTelemetry Collector](/docs/collector/), to a backend such as
 [Jaeger](https://jaegertracing.io/), [Zipkin](https://zipkin.io/),
-[Prometheus](https://prometheus.io/) or a [vendor-specific](/ecosystem/vendors/)
-one.
+[Prometheus](https://prometheus.io/), or a
+[vendor-specific](/ecosystem/vendors/) backend.
+
+This page covers the main OpenTelemetry {{ $name }} exporters and how to set
+them up.
+
+## Available exporters
+
+The registry contains the [list of exporters for {{ $name }}][reg].
 
-As part of {{ $name }} you will find many exporters being available. Among them,
-the OpenTelemetry Protocol (OTLP) exporters provide the best experience for you
-as an end-user, since it is a general-purpose telemetry data delivery protocol
-designed in the scope of the OpenTelemetry project.
+Among exporters, [OpenTelemetry Protocol (OTLP)][OTLP] exporters provide an
+optimal experience since they are lossless and support all [signals]. To learn
```

Comment:
"lossless" needs some clarification... It could be misinterpreted as "100% reliable".

Reply:
Hmm, you're right. Ok, I'll try to think of something better, but it'll likely have to wait a couple of days, I need to shift projects for a bit.

If someone has other suggested wording, suggestions are most welcome. (@cartermp has some good ideas :))

Reply:
OTLP Exporters are designed with OTel data model (link to it) in mind, allowing it to carry OTel data model without any loss of information. (Other exporters may have information loss, like some exporters may not have a good place to store resources, instrumentation scope etc. but OTLP carry everything).

Reply:
Here's a proposed phrasing based largely on @cijothomas's words:

> Among exporters, [OpenTelemetry Protocol (OTLP)][OTLP] exporters are
designed to carry OpenTelemetry data without any loss of information. Furthermore,
there is an abundance of tools that operate on telemetry and support OTLP,
allowing you to a high degree of flexibility when you need it. To learn
more about the OTLP protocol, see [OTLP Specification][OTLP].

Reply:
You can also call out that, some tools like [Jaeger ](https://opentelemetry.io/blog/2022/jaeger-native-otlp/) and [Prometheus ](https://prometheus.io/docs/prometheus/latest/feature_flags/#otlp-receiver) now natively accept OTLP.

Reply:
I think that's a very good point, not only jaeger and prometheus but all the vendors on our vendor page.

Reply:
I've incorporated all of your feedback and revisited the paragraph text. PTAL.